zerver/openapi/javascript_examples.js

```diff
@@ -287,6 +286,7 @@ add_example('update_message_flags', '/messages/flags:post', 200, async (client)
     // Add the "read" flag to the messages with IDs in "message_ids"
     const addflag = {
         messages: message_ids,
+        op: 'add',
```

Comment:
These `op` changes are unnecessary.

Reply:
(And I think this whole commit is wrong and should be dropped; we already merged the correct version).  

You just need to replace this with a commit fixing the last unmigrated file in `templates/zerver/api/get-events.md`.

Reply:
@timabbott Actually, I just rebased the branch and pushed (and didn't look too closely what was getting changed.). I was yet to start on moving the last example from `get-events.md`.